src/main/java/org/folio/circulation/domain/policy/RequestPolicy.java

```diff
@@ -0,0 +1,41 @@
+package org.folio.circulation.domain.policy;
+
+import java.util.ArrayList;
+
+import org.folio.circulation.domain.RequestType;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+public class RequestPolicy {
+
+  private ArrayList<String> requestTypes;
+  private final JsonObject representation;
+
+  public RequestPolicy(JsonObject representation){
+    this.representation = representation;
+    populateRequestTypes();
```

Comment:
This method could be replaced with use of the JsonStringArrayHelper (have not statically imported to make the usage clearer)
```suggestion
    this.requestTypes = JsonStringArrayHelper
      .toStream(representation, "requestTypes")
      .collect(Collectors.toList());
```

Reply:
Thanks for the suggestion. Replaced.